### ISSUE TITLE
EkstraBladet.dk - Embedded videos blocked (EasyList Cookie List / Fanboy's Annoyance List) [GDPR]

### ISSUE THREAD

**Issue by THEtomaso** [ryanbr/fanboy-adblock#1410]
### Affected site:
`https://ekstrabladet.dk/`

### Issue:
Embedded videos are missing (grey frames), because of a blocked cookie consent, which needs to be accepted!

### Example:
`https://ekstrabladet.dk/flash/dkkendte/paa-hospitalet-igen-det-er-vildt-haardt/8292079`

### Screenshot:
Not needed, as the problem is obvious.

### Problem filters:
* **EasyList Cookie List** (a.k.a. Fanboy's Cookie List)
* **Fanboy's Annoyance List**

### Problem rule:
`||consent.dk^$third-party
`
### Possible fix:
`@@||consent.dk^$third-party,domain=ekstrabladet.dk
`
### Reference:
https://github.com/easylist/easylist/issues/6067#issuecomment-694133296

--

### System info:
* OS/version: Windows 8.1 Pro (x64)
* Browser/version: Pale Moon v28.13.0 (x64)
* Adblock Extension/version: uBlock Origin v1.16.4.25 (XUL)

**Comment by THEtomaso** [comment]
_First post edited, due to site changes._

**Comment by Yuki2718** [comment]
I'm not sure what video is missing.

**Comment by THEtomaso** [comment]
_Screenshot added to first post_

**Comment by Yuki2718** [comment]
Yep, I was just looking different (or top) page. Fixed for AdGuard, but set-cookie is not available to uBO so user has to manually accept consent.

**Comment by DandelionSprout** [comment]
I seem to have encountered the same problem too and with I Don't Care About Cookies, but only recently, which I had mistaken for geolocking.

I'm honestly unsure what the best approach would be to fix it, given how extensive the site's cookie prompts are.

**Comment by THEtomaso** [comment]
> I'm honestly unsure what the best approach would be to fix it

In the case of **IDCAC**, simply adding `|~ekstrabladet.dk` to the problem rule (which I included in the first post, under "Other problem filters") will fix the issue.

As for the remaining problem filters (**EasyList Cookie List** / **Fanboy's Annoyance List** and **Fanboy's Enhanced Tracking List**); I've included suggestions for working fixes for them too, in the first post.

**Comment by kiboke** [comment]
@THEtomaso thank you, it will be fixed in IDCAC today.
